x-pack/plugins/synthetics/common/formatters/browser/formatters.ts

```diff
@@ -18,28 +18,19 @@ import { tlsFormatters } from '../tls/formatters';
 export type BrowserFormatMap = Record<keyof BrowserFields, Formatter>;
 
 const throttlingFormatter: Formatter = (fields) => {
-  if (!fields[ConfigKey.IS_THROTTLING_ENABLED]) return 'false';
+  const throttling = fields[ConfigKey.THROTTLING_CONFIG];
 
-  const getThrottlingValue = (v: string | undefined, suffix: 'd' | 'u' | 'l') =>
-    v !== '' && v !== undefined ? `${v}${suffix}` : null;
+  if (!throttling || throttling?.label === 'no-throttling') {
+    return 'false';
+  }
 
-  return [
-    getThrottlingValue(fields[ConfigKey.DOWNLOAD_SPEED], 'd'),
-    getThrottlingValue(fields[ConfigKey.UPLOAD_SPEED], 'u'),
-    getThrottlingValue(fields[ConfigKey.LATENCY], 'l'),
-  ]
-    .filter((v) => v !== null)
-    .join('/');
+  return `${throttling.download}d/${throttling.upload}u/${throttling.latency}l`;
```

Comment:
Not sure if this formatter is even in use? When looking at the monitors sent directly to the service, I can see that it's the right shape, the object rather than the string.